src/components/toogle-switch/toggle-switch.tsx

```diff
@@ -1,11 +1,7 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { color_canvas_primary_inverted } from "../../constants/colors";
 
-const Switch = styled.div`
-  position: relative;
-  width: 28px;
-  height: 18px;
-`;
+const color_control_disabled = "#808080";
```

Comment:
Move to colors.ts. I would name it `dim_canvas_tertiary`. Make sure it looks good on the light theme

Reply:
Done.